.gitignore

```diff
@@ -1,36 +0,0 @@
-# Compiled class file
```

Comment:
Any reason for deleting this file?

Reply:
This was deleted automatically when gh-pages was deployed.

Reply:
Okay